src/API/index.js

```diff
@@ -1,87 +1,91 @@
-import axios from 'axios';
+const protocol = 'http';
+const host = process.env.ROBOKACHE_HOST || 'lvh.me';
+const port = 8080;
+
+/**
+ * URL Maker
+ * @param {string} ext extension to append to url
+ */
+const base_url = `${protocol}://${host}:${port}/api/`;
+
+// Base request method for all endpoints
+async function baseRequest(path, method, body, auth) {
+  let config = {
+    method: method,
+    body: body && JSON.stringify(body),
+    credentials: "include",
+    headers: {
+      'Accept': 'application/json',
+      'Content-Type': 'application/json',
+    },
+  };
+  if(auth)
+    config.headers.Authorization = `Bearer ${auth}`;
```

Comment:
I think there should be two ways to write an IF statement, a one-liner or my preferred option, with {}. I think this is just hard to read.
Also, please put a space after the if. I'm not a fan of auto-linting, but we might could look into some more specific linting rules.